qiskit/pulse/commands/snapshot.py

```diff
@@ -57,8 +69,9 @@ def __eq__(self, other):
             bool: are self and other equal.
         """
         if (type(self) is type(other) and
-                self.name == other.name and
-                self.type == other.type):
+                self.label == other.label and
+                self.type == other.type and
+                self.name == other.name):
```

Comment:
I do not think we need to include `name` in the equality check as this parameter is just used for display purposes.